live-examples/html-examples/embedded-content/iframe.html

```diff
@@ -0,0 +1,8 @@
+<iframe id="inlineFrameExample"
+    title="Inline Frame Example"
+    width="360"
+    height="240"
+    src="https://www.openstreetmap.org/export/embed.html?bbox=-0.004017949104309083%2C51.47612752641776%2C0.00030577182769775396%2C51.478569861898606&amp;layer=mapnik">
+</iframe>
+
+<footer><a href="https://www.openstreetmap.org">Map</a> by OpenStreetMap Foundation (OSMF)</footer>
```

Comment:
I don't think we need the footer here, as the embedded OSM includes their copyright statement (they are quite particular about copyright: https://www.openstreetmap.org/copyright).

But I would also welcome @schalkneethling 's opinion here.

If we can omit the footer then of course we can also remove the footer CSS and also the `position: relative` statement.

Reply:
@wbamberg @sheeeng I agree, we do not need the additional footer here.

Reply:
Footer removed.